server/src/GraphqlHTTPRequest.ts

```diff
@@ -1,10 +1,31 @@
 import { DocumentNode } from 'graphql';
 
 import { httpRequest } from './HTTPRequest';
+import { UPDATE_INVESTIGATION_METADATA } from './DBService/InvestigationMetadata';
 
 export const graphqlURL = '/coronai/graphql';
 export const baseUrl = 'http://localhost:';
 
-export const graphqlRequest = (query: DocumentNode, variables?: any) => (
+export const graphqlRequest = (query: DocumentNode, requestHeaders: any, variables?: any) => (
     httpRequest(baseUrl+process.env.PORT+graphqlURL, 'POST', {query: query.loc?.source.body, variables: { ...variables}})
+    .then((result) => {
+        if (requestHeaders) {
+            //@ts-ignore
```

Comment:
why do you need this?

Reply:
השדה defenitions[0] לא מוגדר היטב אז TS מעיר שאין שדה operation למרות שיש